app/components-react/pages/Loader.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from 'react';
+import Svg from 'components-react/shared/Svg';
+
+const loadingStrings = [
+  'Initiating doggo petting engine...',
+  'Compiling food pic library...',
+  'Calibrating RNG blame threshold...',
+  'Dropping fire mixtape...',
+  'Reticulating emoji splines...',
+  'Waking up 24/7 support team...',
+  'Asking the stream gods for perfect settings...',
+  'Powering up the clouds...',
+  'Constructing additional widgets...',
+  "Remembering all your chatters' names...",
+  'Executing good vibes protocol...',
+  'Submitting mod thank-you notes...',
+  'Expanding audience reach with multistream...',
+  'Crafting perfect VOD with selective recording...',
+  'Calculating multistream hype multiplier coefficient...',
+];
+
+export default function Loader() {
+  const [loaderText, setLoaderText] = useState('');
+  useEffect(lifecycle, []);
+
+  function lifecycle() {
+    function loopRandomText() {
+      const randomIndex = Math.floor(Math.random() * loadingStrings.length);
+      if (loaderText === loadingStrings[randomIndex]) {
+        loopRandomText();
+      } else {
+        setLoaderText(loadingStrings[randomIndex]);
+      }
+    }
+    loopRandomText();
+    const interval = setInterval(loopRandomText, 4000);
+
+    return function cleanup() {
+      clearInterval(interval);
+    };
+  }
+
+  return (
+    <div className="s-loader">
+      <div className="s-loader__bg">
+        <div className="s-loader__inner">
+          <Spinner />
+          <div className="s-loader__text">{loaderText}</div>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function Spinner() {
+  return (
+    <div className="s-spinner s-spinner__overlay">
+      <div className="s-bars">
+        <Svg src={spinnerSrc} className="s-spinner--large" />
```

Comment:
Maybe change it to something like SvgContainer? `Svg` is already a reserved tag and IDE will not automatically suggest you importing it